man/systemd-socket-proxyd.xml

```diff
@@ -85,6 +85,12 @@
     <variablelist>
       <xi:include href="standard-options.xml" xpointer="help" />
       <xi:include href="standard-options.xml" xpointer="version" />
+      <varlistentry>
+        <term><option>--connections-max</option></term>
+        <term><option>-c</option></term>
+
+        <listitem><para>Sets the maximum number of simultaneous connections.  The default is 256.</para></listitem>
```

Comment:
I think this should also say: "If the limit of concurrent connections is reached further connections will be refused." or something like that.